src/helpers.js

```diff
@@ -1,38 +1,68 @@
 'use babel'
 
-import ChildProcess from 'child_process'
-import { createFromProcess } from 'process-communication'
 import { join } from 'path'
 import escapeHTML from 'escape-html'
 import ruleURI from 'eslint-rule-documentation'
 import { generateRange } from 'atom-linter'
+import cryptoRandomString from 'crypto-random-string'
```

Comment:
Just curious, why did you choose this package over, say, https://github.com/kelektiv/node-uuid?